examples/symcache_debug.rs

```diff
@@ -113,37 +117,53 @@ fn execute(matches: &ArgMatches) -> Result<(), Box<Error>> {
 fn main() {
     let matches = App::new("symcache-debug")
         .about("Works with symbol files with the symcache interface")
```

Comment:
does noboy like my indentation style for defining clap apps? :'(

Reply:
`rust-fmt` :/